packages/esm-ward-app/src/types/index.ts

```diff
@@ -96,34 +98,14 @@ export interface InpatientRequest {
 
 export type DispositionType = 'ADMIT' | 'TRANSFER' | 'DISCHARGE';
 
-// InpatientRequestOld[] returned by:
-//    GET /rest/emrapi/inpatient/visits
-// It is also returned by the following endpoints which have been deprecated:
-//    GET /rest/emrapi/inpatient/admissionRequests
-//    GET /rest/emrapi/inpatient/transferRequests
-//    GET /rest/emrapi/inpatient/admissionAndTransferRequests
-export interface InpatientRequestOld {
-  patient: Patient;
-  visit: Visit;
-  type: DispositionTypeOld;
-
-  // as of now, these fields are not included in the backend
-  encounter?: Encounter;
-  dispositionObs?: Observation;
-  dispositionLocation?: Location;
-  dispositionDate?: Date;
+// GET /rest/emrapi/inpatient/visits
+export interface InpatientAdmissionFetchResponse {
+  results: InpatientAdmission[];
 }
 
-export type DispositionTypeOld = 'ADMISSION' | 'TRANSFER' | 'DISCHARGE';
-
-// AdmittedPatient[] returned by:
-// GET /rest/emrapi/inpatient/visits
-export interface AdmittedPatient {
+export interface InpatientAdmission {
   patient: Patient;
   visit: Visit;
```

Comment:
Will need to add properties like "latestAdmissionOrTransferEncounter" once we modify the REP to return them.

Reply:
I'm confused.  You are setting a custom rep already in this PR - why aren't you including that property?

Reply:
I was just starting with the basics and noted that the new property will be needed in the follow-on ticket I created.

Reply:
Seems odd though to make a comment elsewhere in the PR that something needs to be added to the rep, when you are defining the rep.  Why not just include it in the first place and not have to make that comment?